listener.py

```diff
@@ -191,87 +236,96 @@ def exitS_regex(self, ctx:APiParser.S_regexContext):
     # Enter a parse tree produced by APiParser#json.
     def enterJson(self, ctx:APiParser.JsonContext):
         try:
-            print( ctx.children[ 0 ].getText() )
+            # print( ctx.children[ 0 ].getText() )
+            pass
         except:
             pass
 
     # Exit a parse tree produced by APiParser#json.
     def exitJson(self, ctx:APiParser.JsonContext):
-        top = self.STACK.pop()
```

Comment:
Commenting out JSON / XML parser semantics because I think it is much more useful to have it in raw format, so we can apply JSON / XML libraries to read the content